adapter-in/web/src/test/kotlin/com/yapp/bol/group/GroupControllerTest.kt

```diff
@@ -69,5 +56,68 @@ class GroupControllerTest : ControllerTest() {
                     )
                 )
         }
+
+        test("GET /v1/group/search") {
+            val name = "뽀글뽀글"
+            val pageNumber = 0
+            val pageSize = 10
+
+            every {
+                groupService.searchGroup(any(), any(), any())
+            } returns ApplicationSlice<GroupWithMemberCount>(
+                content = listOf(GROUP_WITH_MEMBER_COUNT),
+                hasNext = false,
+            )
+
+            get("/v1/group/search?name=$name&pageNumber=$pageNumber&pageSize=$pageSize") {}
+                .isStatus(200)
+                .makeDocument(
+                    DocumentInfo(identifier = "searchGroup", tag = OpenApiTag.GROUP),
+                    queryParameters(
+                        "name" type STRING means "그룹 이름",
+                        "pageNumber" type NUMBER means "페이지 번호",
+                        "pageSize" type NUMBER means "페이지 크기",
+                    ),
+                    responseFields(
+                        "content" type ARRAY means "그룹 목록",
+                        "content[].id" type NUMBER means "그룹 ID",
+                        "content[].name" type STRING means "그룹 이름",
+                        "content[].description" type STRING means "그룹 설명",
+                        "content[].organization" type STRING means "그룹 소속",

```

Comment:
p3. 저희 소속은 nullable 로 가는 걸로 기억해요.  생각해보니 여태 Group 소속 NotNull로 처리한 것 같네요

```suggestion
                        "content[].organization" type STRING means "그룹 소속" isOptional true,

```

Reply:
> p3. 저희 소속은 nullable 로 가는 걸로 기억해요. 생각해보니 여태 Group 소속 NotNull로 처리한 것 같네요

그렇네요~ 요거 지금까지는 아마 클라에서 빈 값으로 처리해줘서 괜찮았던거 같은데, nullable 로 수정해두겠습니다!

Reply:
> p3. 저희 소속은 nullable 로 가는 걸로 기억해요. 생각해보니 여태 Group 소속 NotNull로 처리한 것 같네요

아래에 수정했습니다
ebd1d8bac0ade255d651b9a6361fd98ab4b4ec4a

컨플릭이 좀 두렵군요..!